### PR TITLE
Fix some compilation errors on MacOS due to an old SDK

### PR DESCRIPTION
**Pull request by amontoison**
#94

### CODE REVIEW

File: .github/julia/build_tarballs_yggdrasil.jl [hunk 1]
```diff
@@ -9,9 +9,12 @@ haskey(ENV, "UNO_URL") || error("The environment variable UNO_URL is not defined
 name = "Uno"
 version = VersionNumber(ENV["UNO_RELEASE"])
 
+
```
Comment (amontoison) [inline .github/julia/build_tarballs_yggdrasil.jl:12]:
```suggestion
```

**Review by cvanaret — APPROVED** [review]

**Comment by cvanaret** [comment]
Thanks @amontoison!
The command `rm -rf /opt/${target}/${target}/sys-root/System` in the `x86_64-apple-darwin14` build fails:

```
[18:02:50] rm: can't remove '/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Printers/Libraries': I/O error
[18:02:50] rm: can't remove '/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Tcl.framework/Versions/8.4/Headers/tcl-private': I/O error
[18:02:50] rm: can't remove '/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/Tcl.framework/Versions/8.5/Headers/tcl-private': I/O error
[18:02:50] rm: can't remove '/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/ForceFeedback.framework/Versions/A/Modules': I/O error
[18:02:50] rm: can't remove '/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/ForceFeedback.framework/Versions/A/Headers': I/O error
[18:02:50] rm: can't remove '/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/QTKit.framework/Versions/A/Modules': I/O error
[18:02:50] rm: can't remove '/opt/x86_64-apple-darwin14/x86_64-apple-darwin14/sys-root/System/Library/Frameworks/QTKit.framework/Versions/A/Headers': I/O error
...
```

**Comment by amontoison** [comment]
Where do you have this error?
Is it in a CI build?

**Comment by cvanaret** [comment]
I retagged to trigger the release build. Perhaps too soon?

**Comment by amontoison** [comment]
It will not work because you have other compilation errors:
https://github.com/cvanaret/Uno/issues/94#issuecomment-2480636662

**Comment by cvanaret** [comment]
Ah sorry, I overlooked that one. Looks like the good old missing include that no compiler ever complained about (except now).

**Pull request merged**